shared/profile/index.js

```diff
@@ -27,67 +35,92 @@ class Profile extends Component<void, Props, void> {
   }
 
   componentDidMount () {
-    this.props.refresh(this.props.username)
+    this.props && this.props.refresh()
   }
 
   componentWillReceiveProps (nextProps) {
-    if (nextProps.username !== this.props.username) {
-      this.props.refresh(nextProps.username)
+    const oldUsername = this.props && this.props.username
+    if (nextProps && nextProps.username !== oldUsername) {
+      nextProps.refresh()
     }
   }
 
   render () {
-    return (
+    return this.props ? (
       <Render
         showComingSoon={!flags.tabProfileEnabled}
         {...this.props}
         proofs={this.props.proofs || []}
         loading={this.props.loading}
         onBack={!this.props.profileIsRoot ? this.props.onBack : undefined}
-        followers={this.props.trackers || []}
-        following={this.props.tracking || []}
       />
+    ) : (
+      <Box><Text type='Error'>Could not derive props; check the log</Text></Box>
```

Comment:
Something like "We had an error displaying this data." might be a little less alarming/dire sounding.